src/matching_market.sol

```diff
@@ -17,8 +17,10 @@
 
 pragma solidity ^0.5.12;
 
-import "./expiring_market.sol";
 import "ds-note/note.sol";
+import "ds-auth/auth.sol";
```

Comment:
Wasn't it to be a contract without an owner? why are we keeping this?
We should remove `setBuyEnabled` and `setMatchingEnabled` as well.